src/components/DonationForm.jsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
```

Comment:
`DonationForm.jsx` and `DonationForm.module.css` should be in their own folder named `DonationForm` within `/components`, see [PNP Best Programming Practices](https://drive.google.com/file/d/1EfQjal0X2wpreBFlGLLqZznisbncYdFf/view) for more info